strawberry/static/graphiql.html

```diff
@@ -1,103 +1,68 @@
+<!--
+    *  Modified from: https://github.com/graphql/graphiql/blob/main/examples/graphiql-cdn/index.html
+-->
 <!DOCTYPE html>
 <html>
 
 <head>
     <title>Strawberry GraphiQL</title>
+    <link rel="shortcut icon" href="https://strawberry.rocks/favicon.ico">
```

Comment:
not sure if we should include our favicon here 🤔

Reply:
mostly because we'd receive requests not really meant to us[1] :)

maybe we can use one of the solutions here to inline the favicon? https://stackoverflow.com/questions/5199902/how-to-save-up-another-precious-http-request-for-the-tiny-favicon

[1] I wonder if we could check the referer of these requests from our logs

Reply:
Yeah that's fair enough, how would you feel about using a strawberry emoji as a favicon?
i.e.
```
<link
        rel="icon"
        href="data:image/svg+xml,
        <svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22>
            <text y=%22.9em%22 font-size=%2280%22>🍓</text>
        </svg>"
    >
```

And I think I should be able to add the explorer, will just require a bit of JS scripting/React

Reply:
emoji sounds great!